python/cuml/test/dask/test_comms.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+from dask_cuda import LocalCUDACluster
+
+import random
+
+import time
+
+from dask.distributed import Client, wait
+
+from cuml.dask.common.comms import CommsContext, worker_state, default_comms
+from cuml.dask.common import perform_test_comms_send_recv
+from cuml.dask.common import perform_test_comms_allreduce
+
+
+def test_comms_init_no_p2p():
+
+    cluster = LocalCUDACluster(threads_per_worker=1)
+    client = Client(cluster)  # noqa
+
+    cb = CommsContext()
+    cb.init()
```

Comment:
set the parameters for no p2p expliclty here, and perhaps add a test for init with default parameters (which might change in the future)

Reply:
Once UCX packaging is complete & UCX is no longer considered experimental, the plan is to remove the `comms_p2p` altogether. I'll add the `comms_p2p=False` for now.